docker/script/entrypoint.sh

```diff
@@ -81,14 +81,35 @@ fi
 
 case "$1" in
   local-runner)
+    # if S3_PLUGINS_PATH
+    if [ -n "$S3_PLUGINS_PATH" ]; then
+      echo "Downloading $S3_PLUGINS_PATH"
+      mkdir -p /usr/local/airflow/plugins
+      cd /usr/local/airflow/plugins
+      aws s3 cp $S3_PLUGINS_PATH plugins.zip
+      unzip -o plugins.zip 
+      rm plugins.zip
+    fi
+    # if S3_DAGS_PATH
+    if [ -n "$S3_DAGS_PATH" ]; then
+      echo "Syncing $S3_DAGS_PATH"   
+      mkdir -p /usr/local/airflow/dags
+      cd /usr/local/airflow/dags
+      aws s3 sync --exact-timestamp $S3_DAGS_PATH .
```

Comment:
Adding a --delete flag to the command might be useful. Ensures un necessary or stale files are deleted during sync. In the absence of the flag the user might happen to have differences between the local and cloud locations and not notice them.

https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html